README.md

```diff
@@ -23,7 +23,7 @@ A curated list of awesome things and projects built by Algerian developers.
 - [Salat Time](https://github.com/Barbarosse/salat_time/) A simple python Web App that returns the five Islamic Prayer Times.
 - [selma](https://github.com/FdevTech/Selma) - A virtual assistant backed by augmented reality capabilities, it allows you to control different appliances in your home.
 - [Timgad](https://github.com/OpenDZ/timgad) - Timgad is a kernel Linux Security Module that collects per process and system-wide security protections that are not handled by the core kernel itself.
-
+- [salati](https://github.com/seif1000/salati) an android app that helps muslims to performe their prayers in time by reminding them every day when the time comes  by adhan alarm .
```

Comment:
Hi There! Thanks for the addition, I saw that your project is missing a LICENSE file, can you fix that?

Also, would you be able to put this on the right alphabetical order spot (after Salat Time)

Reply:
I fixed it  .